apis/field_error_test.go

```diff
@@ -503,6 +616,113 @@ func TestFlatten(t *testing.T) {
 	}
 }
 
+func TestFieldErrorPerformance(t *testing.T) {
+	limit := int64(3) // limit performance decrease to at most 3x slower with merge
+	count := 100
+
+	var withoutMerge time.Duration
+	var withMerge time.Duration
+
+	{
```

Comment:
This is weird (though I used to do this in C++ commenting scopes of code), use `t.Run` to describe the scopes?